recipes/pdf_reports/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "pdf_reports" %}
+{% set version = "0.3.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Edinburgh-Genome-Foundry/pdf_reports/archive/refs/tags/v{{ version }}.tar.gz
```

Comment:
This package is on PyPI. Seems sufficiently generic to be a Conda Forge package instead.